fwprovider/cluster/metrics/resource_metrics_server_test.go

```diff
@@ -165,6 +171,51 @@ func TestAccResourceMetricsServer(t *testing.T) {
 				),
 			},
 		}},
+		{"create opentelemetry metrics server & import it", []resource.TestStep{
+			{
+				ResourceName: "proxmox_virtual_environment_metrics_server.acc_otel_server",
```

Comment:
Thanks for adding the tests!

Unfortunately, I can't keep them enabled, as apparently Proxmox is trying to connect to OTel backend when it creates the resource. So without `192.168.3.2`:4318 server deployed the test is always failing.

Would you mind adding something like this to each new test step you've created?
```suggestion
			{
				// Skip this test until we have a way to test opentelemetry servers (i.e. setting up local otel collector)
				// Proxmox is trying to connect to the server when creating the resource.
				SkipFunc: func() (bool, error) {
					return true, nil
				},
				ResourceName: "proxmox_virtual_environment_metrics_server.acc_otel_server",
```